
<launch>

  <rosparam  command="load"  file="$(find laser_mapper)/param/canyon_mapper_output.yaml"/>

  <!--include file="$(find laser_stitcher)/launch/arbotix.launch"> </include-->

   <node
      name    = "laser_mapper"
      pkg     = "laser_mapper"
      type    = "laser_mapper"
      args    = ""
      output  = "screen"
    >
  </node>

  <node
      name    = "pointcloud_processing_server"
      pkg     = "pointcloud_processing_server"
      type    = "pointcloud_processing_server"
      args    = "-intensity"
      output  = "log"
    >
  </node>

</launch> 